splinterd/tests/framework/network.rs

```diff
@@ -87,6 +87,13 @@ impl Network {
             auth_types: vec![AuthType::Biome, AuthType::Cylinder],
         }
     }
+    pub fn new_with_auth(auth_types: Vec<AuthType>) -> Self {
```

Comment:
There could be a method that takes `&mut self` to set these values.

Reply:
removed